eox_nelp/pearson_vue/models.py

```diff
@@ -47,3 +49,4 @@ class PearsonRTENEvent(models.Model):
     content = models.JSONField()
     created_at = models.DateTimeField(auto_now_add=True)
     event_type = models.CharField(max_length=20, choices=EVENT_TYPE_CHOICES)
+    candidate = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
```

Comment:
Do you think we could use `blank=True`, in order to don't be mandatory in forms or serializer.?

Reply:
I wanted to do this as mandatory as possible since the idea is that every record has a user however we don't know if the client will send the whole data in the example format and secondly the migration process complains about previous records when the null=True field is not present, that's why I added the null field and I don't think we should include the blank field